api/v1beta1/grafanacontactpoint_types_test.go

```diff
@@ -41,7 +41,7 @@ var _ = Describe("ContactPoint type", func() {
 
 		It("Should block adding name field when missing", func() {
 			contactpoint := newContactPoint("adding-name")
-			contactpoint.Spec.Type = "webhook" // nolint:goconst
+			contactpoint.Spec.Type = "webhook" //nolint:goconst
```

Comment:
Looks like one got through
```suggestion
			contactpoint.Spec.Type = "webhook"
```